packages/paste-core/components/breadcrumb/stories/index.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+storiesOf('Components|Breadcrumb', module)
+  .addDecorator(withKnobs)
+  .add('Anchors', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Anchor href="#">Active Numbers</Anchor>
+        </BreadcrumbItem>
+      </Breadcrumb>
+    );
+  })
+  .add('Anchors and current page ', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Anchor href="#">Active Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Text aria-current="page" as="span" fontSize="fontSize20" lineHeight="lineHeight20">
+            Infrrd Germany hotline B
+          </Text>
+        </BreadcrumbItem>
+      </Breadcrumb>
+    );
+  })
+  .add('Anchors with truncation ', () => {
+    return (
+      <Breadcrumb>
+        <BreadcrumbItem>
+          <Anchor href="#">Phone Numbers</Anchor>
+        </BreadcrumbItem>
+        <BreadcrumbItem>
+          <Box as="span" display="inline-flex" maxWidth="size10">
+            <Truncate title="Some very long text to truncate">
```

Comment:
An inline-flex Box with a Truncate child seems rather complex.  Can we hide this complexity behind a prop? Do we have any opinions on maxWidth?

Reply:
Truncation is a little tricky here. I don't think we should have strong opinions about when text is truncated until we get feedback from users, especially in an alpha version.

The width may also need some research/testing on the design side. I'm sure there are some opinions there.

Maybe we just show guidance here for the alpha version. The new API does simplify it a little:
```
<BreadcrumbItem href="#">
    <Box as="span" display="inline-flex" maxWidth="size10">
        <Truncate title="Some very long text to truncate">Some very long text to truncate</Truncate>
    </Box>
</BreadcrumbItem>
```

Reply:
Can we make `maxWidth` a prop on `BreadcrumbItem`, and if passed it does the truncation automagically?